doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part1.ipynb

```diff
@@ -9,11 +9,12 @@
     "#### Before you start:\n",
```

Comment:
<blockquote>After almost 20 years of developmentI</blockquote><p>I disagree, in 2020 we have more than 30 years of development: cellular automata were popular in the 1980s and Succi's first paper with ther term "Lattice Boltzmann" is from 1989, not sure if this was the first paper which introduced the name though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>good point, thanks!</p><p> </p>